mayastor/src/bdev/nvmx/handle.rs

```diff
@@ -1045,6 +1096,13 @@ impl BlockDeviceHandle for NvmeDeviceHandle {
             )
         };
 
+        if rc < 0 {
+            return Err(CoreError::NvmeIoPassthruDispatch {
+                source: Errno::from_i32(rc.abs()),
```

Comment:
Can you make use of the `to_result()` from ffiresult.rs?

Reply:
Sure, I'd forgotten that that exists.